app/_includes/md/gateway/deployment-topologies.md

```diff
@@ -0,0 +1,118 @@
+{% if include.topology == "konnect" %}
+<!--vale off -->
+{% mermaid %}
+flowchart TD
+A(Dev Portal &bull; Gateway Manager &bull; Analytics &bull; Service Hub)
+B(<img src="/assets/images/logos/kogo-white.svg" style="max-height:20px" class="no-image-expand"/> Control plane \n #40;{{site.base_gateway}} instance#41;)
+B2(<img src="/assets/images/logos/kogo-white.svg" style="max-height:20px" class="no-image-expand"/> Control plane \n #40;{{site.base_gateway}} instance#41;)
+C(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Data plane 3\n #40;{{site.base_gateway}} instance#41;)
+D(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Data plane 1\n #40;{{site.base_gateway}} instance#41;)
+E(<img src="/assets/images/logos/KogoBlue.svg" style="max-height:20px" class="no-image-expand"/> Data plane 2\n #40;{{site.base_gateway}} instance#41;)
+
+subgraph id1 [Konnect]
+A --- B & B2
+end
+
+id1 --Kong proxy 
+configuration---> C & D & E
+
+subgraph id2 [Self-managed on-premise node]
```

Comment:
Is it possible to fix this arrow?
![image](https://github.com/Kong/docs.konghq.com/assets/23319190/05dec74e-2eda-45b7-9ebf-2b1e5cc802de)